src/pages/Dashboard/Project-Selector.vue

```diff
@@ -182,10 +180,25 @@ export default {
   color: #3b8dff;
   font-weight: 500;
 }
+</style>
 
+<style lang="scss">
 .project-selector {
-  font-size: 0.9rem;
+  font-size: 16px;
```

Comment:
another 'out of curiosity' -- why switch from rems to px?

Reply:
This was swapped to match the vuetify `v-select` that we use for versions on the flow groups page - the CSS inheritance for this particular component meant we'd have to change upstream classes as well to match the context of the `v-select` OR explicitly set it here... for simplicity I chose the latter.